src/display/editor/annotation_editor_layer.js

```diff
@@ -234,11 +259,26 @@ class AnnotationEditorLayer {
 
   attach(editor) {
     this.#editors.set(editor.id, editor);
+    if (
+      editor.annotationElementId &&
+      this.#uiManager.isDeletedAnnotationElement(editor.annotationElementId)
+    ) {
+      this.#uiManager.removeDeletedAnnotationElement(
+        editor.annotationElementId
+      );
+      editor.deleted = false;
```

Comment:
Would it perhaps make sense, and be less error-prone, if we simply pass the `editor` to the `removeDeletedAnnotationElement`-method instead?
That way you could set the `editor.deleted = false;` line in that method and we'd not have to worry about remembering to do that manually here.